storage-resize-images/functions/src/resize-image.ts

```diff
@@ -96,14 +103,14 @@ export const modifyImage = async ({
   contentType: string;
   size: string;
   objectMetadata: ObjectMetadata;
+  format: string;
 }): Promise<ResizedImageResult> => {
-  const { imageType } = config;
-  const hasImageTypeConfigSet = imageType !== "false";
-  const imageContentType = hasImageTypeConfigSet
-    ? supportedImageContentTypeMap[imageType]
+  const useOriginalFormat = format !== "original";
```

Comment:
also, the naming would be something like `notUsingOriginalFormat`

Reply:
Renamed and changed to `false`.

Can we provide a use case for why this would need to be backwards compatible, the configuration will now include the word `false` which seems a little counter intuitive?

Ideally I think we should continue to keep it as `original` or another named equivalent.

Reply:
The previous versions of this extension used "false" to stop the extension from converting the type. If those users upgrade to this future version via the UI, that config value will no longer stop conversion if it is changed to "original".